pkgs/applications/networking/cluster/kops/default.nix

```diff
@@ -58,6 +58,6 @@ in rec {
 
   kops_1_16 = mkKops {
     version = "1.16.1";
-    sha256 = "08vy57ln0qar961sf9vbrh29f04qj5siqsfim1kqwvj5xrvwa39i";
+    sha256 = "04ji1703z6b3rd4wmccah68api0jj2bndzkgdzqi855gccqbmi5g";
```

Comment:
I am confused. Ofborg disagrees:

```
  wanted: sha256:04ji1703z6b3rd4wmccah68api0jj2bndzkgdzqi855gccqbmi5g
  got:    sha256:08vy57ln0qar961sf9vbrh29f04qj5siqsfim1kqwvj5xrvwa39i
```

Reply:
It builds locally though?

Reply:
I did replace with all-zeros locally and TOFU-copied in the hash I got locally. Seems this is also [what hydra sees](https://hydra.nixos.org/build/117730286/nixlog/1), so I'd assume ofborg is wrong here.

Can you checkout locally too?

Reply:
It built locally as I said.